Hw1/task3.py

```diff
@@ -0,0 +1,22 @@
+Digit = int
+divider = 0
+Start_digit = 2
+One = 1
+
+while True:
+    Digit =int(input("Введите положительное число, которое не больше 100 000: "))
+    if Digit >= 0 or Digit < 100000:
+        break
+
+sqrt = int(Digit ** 0.5)
+
+for i in range(Start_digit, sqrt+One // Start_digit+One):
```

Comment:
Не совсем понятен второй аргумент.
```suggestion
for i in range(Start_digit, sqrt + One):
```